examples/article-parallax.amp.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP Article with parallax title</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <style amp-custom>
+    article, header h1 {
+      margin: 0px 10px;
+    }
+
+    header h1 {
+      position: absolute;
+      top: 25vh;
+      padding: 5px;
+      z-index: 1;
+      max-width: 70vw;
+    }
+
+    header h1 span {
+      background-color: black;
+      color: white;
+      line-height: 1.2em;
+    }
+
+    header amp-img img {
+      object-fit: cover;
+    }
+
+    /*
+     * Vertically center the tite within the image.
+     */
+    .vertically-center {
+      transform: translateY(-50%);
+    }
+  </style>
+<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style>
+<noscript>
+  <style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style>
+</noscript>
+<script async custom-element="amp-fx-parallax" src="../../dist/v0/amp-fx-parallax-0.1.max.js"></script>
```

Comment:
update the paths to point to CDN (to test locally, hit `article-parallax.amp.max.html` (note `.max.`) and server will rewrite the paths to local files.